sample-apps/payment-customization-hide/extensions/hide-first-payment/src/main.rs

```diff
@@ -8,16 +8,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     eprintln!("Hello World");
 
     // read from stdin and write to stdout
```

Comment:
Do we need to print hello world and have these comments?

Reply:
We don't 😛 I'll remove